test/OrchardCoreContrib.PoExtractor.Tests/OrchardCoreContrib.PoExtractor.Tests.csproj

```diff
@@ -25,4 +25,10 @@
     <ProjectReference Include="..\..\src\OrchardCoreContrib.PoExtractor\OrchardCoreContrib.PoExtractor.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Update="ProjectFiles\StringLocalizerExample.cs">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
Do we really need this?

Reply:
The cs file needs to be copied to the output directory because the extractor works against the raw file, not the compiled dll.

Reply:
Specifically [here](https://github.com/OrchardCoreContrib/OrchardCoreContrib.PoExtractor/blob/e30033afb74bf94a2028addf375fe1ceeaad2d56/src/OrchardCoreContrib.PoExtractor.DotNet.CS/CSharpProjectProcessor.cs#L47) where it just looks at the files under the project folder.

Reply:
You 're right, I forgot they need to be copied